README.md

```diff
@@ -2409,6 +2412,15 @@ Optics Letters, 2021.
 *[Shack-Hartmann wavefront sensing using spatial-temporal data from an event-based image sensor](https://doi.org/10.1364/OE.409682)*,  
 Optics Express, 2020.
 
+<a name="super_resolution_imaging"></a>
+### Super-resolution imaging
+- <a name="Cabriel22bioArxiv"></a>Cabriel, C., Specht, C. G. & Izeddin, I.,  
+*[Event-based vision sensor enables fast and dense single-molecule localization microscopy](https://doi.org/10.1101/2022.07.22.501162)*,  
+bioArxiv, 2022.
+- <a name="Mangalwedhekar23NN"></a>Mangalwedhekar, R., Singh, N., Thakur, C. S., Seelamantula, C. S., Jose, M., and Nair, D.,  
```

Comment:
Mangalwedhekar23natnano ? (so that nn is not confused with Elsevier's Neural Networks journal)

Reply:
Done